### PR TITLE
fix get_rook_moves

### PR DESCRIPTION
**Pull request by shashirama**

### CODE REVIEW

File: engine.py [hunk 1]
```diff
@@ -3,243 +3,491 @@
...
+            ["rl", "nl", "bl", "ql", "kl", "bl", "nl", "rl"]]
+
+        self.light_to_move = True # True = light's turn to play; False = dark's turn to play
+        self.move_log = []        # keeps a log of all moves made withing a game
+        self.move_piece = {"p":self.get_pawn_moves, "r":self.get_rook_moves, \
+                        "q":self.get_queen_moves, "k":self.get_king_moves, \
+                        "b":self.get_bishop_moves, "n":self.get_knight_moves}
+
+
+    def get_pawn_moves(self, r, c, moves):
```
Comment (drak00) [inline engine.py:42]: please return this to its default

File: engine.py [hunk 2]
```diff
@@ -3,243 +3,491 @@
...
+        ##TODO
+        pass
+
+
+    def get_knight_moves(self, r, c, moves):
+        ##TODO
+        pass
+
+
+    def get_king_moves(self, r, c, moves):
```
Comment (drak00) [inline engine.py:143]: I'm positive you were not given get_king_moves() task. This should be left empty for your teammates

**Review by drak00 — CHANGES_REQUESTED** [review]
Rook moves as expected!

File: engine.py (hunk 2 above)
Comment (shashirama) [reply engine.py:143]: I'm sorry, I added them so I can test the rook moves properly I will change them and push again

**Pull request closed without merging**